pandas/_libs/src/parser/tokenizer.c

```diff
@@ -1554,6 +1554,7 @@ double xstrtod(const char *str, char **endptr, char decimal, char sci,
     int n;
     int num_digits;
     int num_decimals;
+    int _maybe_int = 1;
```

Comment:
I find this kind of indirection extra confusing, pls don't do this.

Reply:
removed extra _maybe_int variable